shared/yarn.lock

```diff
@@ -12772,11 +12772,17 @@ react-native-safe-module@^1.1.0:
   dependencies:
     dedent "^0.6.0"
 
-react-native-screens@1.0.0-alpha.22, "react-native-screens@^1.0.0 || ^1.0.0-alpha":
+"react-native-screens@^1.0.0 || ^1.0.0-alpha":
   version "1.0.0-alpha.22"
```

Comment:
why does this still exist

Reply:
Dunno, I `yarn install`ed and this stayed. is there a better command to update the lockfile?

Reply:
lets use resolutions to force it over